model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -125,6 +126,11 @@ data class ResolvedLicense(
      */
     val originalDeclaredLicenses: Set<String>,
 
+    /**
+     * The original SPDX expressions of this license from all license sources.
+     */
+    val originalExpressions: Set<SpdxExpression>,
```

Comment:
I was thinking if this should rather be a `Map<LicenseSource, Set<SpdxExpression>>`, this could be relevant to be able to support `LicenseView`s, for example in rules.

Reply:
I agree with your suggestion.

Would this also mean to remove `val sources: Set<LicenseSource>` as this information would be contained in the `Map<LicenseSource, Set<SpdxExpression>>`, hence be duplicated information?
As such this would be a breaking change.

Reply:
I'd like to keep `sources` as it is useful in the API, but if it is certain that `sources` and `originalExpressions.keys` are identical `sources` could be moved from the constructor to the class body as `val sources = originalExpressions.keys`.

Reply:
Removed sources from the constructor and added it to `LicenseView`.